src/utils.ts

```diff
@@ -1,59 +1,28 @@
-import {
-  base64_variants,
-  from_base64,
-  from_hex,
-  from_string,
-  ready,
-  to_base64,
-  to_hex,
-  to_string,
-} from './libsodium'
+import {bytesToHex, randomBytes} from '@noble/hashes/utils'
+import {base64} from '@scure/base'
 
-const BASE64_VARIANT = base64_variants.ORIGINAL
-
-export const hexStringToArrayBuffer = async (hex: string): Promise<Uint8Array> => {
-  await ready
-  return from_hex(hex)
-}
-
-export async function arrayBufferToBase64(arrayBuffer: Uint8Array): Promise<string> {
-  await ready
-  return to_base64(arrayBuffer, BASE64_VARIANT)
-}
-
-export async function base64ToArrayBuffer(base64: string): Promise<Uint8Array> {
-  await ready
-  return from_base64(base64, BASE64_VARIANT)
-}
-
-export async function stringToArrayBuffer(string: string): Promise<Uint8Array> {
-  await ready
-  return from_string(string)
+export function arrayBufferToBase64(arrayBuffer: Uint8Array): string {
+  return base64.encode(arrayBuffer)
 }
 
-export async function arrayBufferToHexString(arrayBuffer: Uint8Array): Promise<string> {
-  await ready
-  return to_hex(arrayBuffer)
+export function base64ToArrayBuffer(b64: string): Uint8Array {
+  return base64.decode(b64)
 }
 
-export async function arrayBufferToString(arrayBuffer: Uint8Array): Promise<string> {
-  await ready
-  return to_string(arrayBuffer)
+export function arrayBufferToString(arrayBuffer: Uint8Array): string {
+  return new TextDecoder().decode(arrayBuffer)
 }
 
-export const generateRandomKey = async (length: number): Promise<string> => {
-  const randomValues = await generateRandomUint8Array(length)
-  return arrayBufferToHexString(randomValues)
+export function generateRandomKey(length: number): string {
+  const randomValues = generateRandomUint8Array(length)
+  return bytesToHex(randomValues)
 }
 
-export const generateRandomUint8Array = async (length: number): Promise<Uint8Array> => {
-  return getRandomValues(new Uint8Array(length))
+export function generateRandomUint8Array(length: number): Uint8Array {
```

Comment:
Is there a need in this method? It's also 1-line alias. randomBytes can be used directly.